tests/tools/test_convert_partition.py

```diff
@@ -0,0 +1,64 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+
+from tools.distpartitioning import utils
+
+from tools.distpartitioning.convert_partition import _get_unique_invidx
+
+
+@pytest.mark.parametrize(
+    "num_nodes, num_edges, nid_begin, nid_end",
+    [
+        [4000, 40000, 0, 1000],
```

Comment:
Can you add test cases as [1,1,0,1], [4000,0,1000,4000], etc. to test the corner cases.

Reply:
Done.

Reply:
[4000, 0, 1000, 4000] cannot be a valid test case, since we always need a non-empty edge-list to process. One of the return parameters of this function is the edge-list itself, which is over-written with the return parameters.